<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V3DCfgWidForAttributeIn3DCartesianGrid</class>
 <widget class="QWidget" name="V3DCfgWidForAttributeIn3DCartesianGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>IJK Clipping</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="2">
        <widget class="QSlider" name="sldKHighClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="sldJLowClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="sldKLowClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSlider" name="sldJHighClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>K (slice) clipping:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>J (cross line) clipping:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>I (inline) clipping:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="sldIHighClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="sldILowClip">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="lblJhighClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lblJlowClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblIhighClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblIlowClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lblKlowClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="lblKhighClip">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Color scaling</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Color table min:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="spinColorMin">
         <property name="minimum">
          <double>-1000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>max:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinColorMax">
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color scaling:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbScaling"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sldIHighClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldILowClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldJHighClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldJLowClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldKHighClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldKLowClip</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinColorMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinColorMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbScaling</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>V3DCfgWidForAttributeIn3DCartesianGrid</receiver>
   <slot>onUserMadeChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUserMadeChanges()</slot>
 </slots>
</ui>
